invenio_circulation/ext.py

```diff
@@ -118,7 +118,6 @@ def trigger(self, loan, **kwargs):
                 t.execute(loan, **kwargs)
                 return loan
             except TransitionConditionsFailedError as ex:
-                current_app.logger.exception(ex)
```

Comment:
It should maybe pass some kind of an exception to the UI?

Reply:
> It should maybe pass some kind of an exception to the UI?

A `NoValidTransitionAvailableError` exception will be raised and displayed in the UI if no transition was found. This would only suppress the logging of transition exceptions that happens before a successful transition has been made.